---
 texmf-dist/dvipdfmx/dvipdfmx.cfg |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

 % Distiller config. Uses ghostscript and works on compressed and
 % uncompressed files.  Use zcat -f <%i instead of zcat -f %i
--- texlive-base-2014.20150305.orig/texmf-dist/dvipdfmx/dvipdfmx.cfg
+++ texlive-base-2014.20150305/texmf-dist/dvipdfmx/dvipdfmx.cfg
@@ -154,7 +154,7 @@
 %% kpsewhich -progname=dvipdfmx -format='other text files' dvipdfmx.cfg
 %% and control which one is found by setting DVIPDFMXINPUTS.
 %%
-D  "rungs -q -dNOPAUSE -dBATCH -dEPSCrop -sPAPERSIZE=a0 -sDEVICE=pdfwrite -dCompatibilityLevel=%v -dAutoFilterGrayImages=false -dGrayImageFilter=/FlateEncode -dAutoFilterColorImages=false -dColorImageFilter=/FlateEncode -sOutputFile='%o' '%i' -c quit"
+D  "gs -q -dNOPAUSE -dBATCH -dEPSCrop -sPAPERSIZE=a0 -sDEVICE=pdfwrite -dCompatibilityLevel=%v -dAutoFilterGrayImages=false -dGrayImageFilter=/FlateEncode -dAutoFilterColorImages=false -dColorImageFilter=/FlateEncode -sOutputFile='%o' '%i' -c quit"
 
 % other random ps converters people have experimented with.
 %D "/usr/local/bin/ps2pdf -dEPSCrop '%i' '%o'"
